web/chat/ChatMessageInput.jsx

```diff
@@ -27,6 +27,11 @@ class ChatMessageInput extends Component {
     this.handleRef = this.handleRef.bind(this);
     this.state = {
       value: '',
+      indicatorProps: {
```

Comment:
I would probably move these settings to defaults on the `ChatMessageLengthIndicator` component so you don't need to set them for our only use case (while still allowing it to be reusable with other settings).